simulation_parameters/microbe_stage/biomes.json

```diff
@@ -312,7 +314,7 @@
             },
             {
               "ScenePath": "res://assets/models/organelles/Nitrogenase.tscn",
-              "ConvexShapePath": "res://assets/models/organelles/Nitrogenase.shape"
+              "ConvexShapePath": "res://assets/models/organelles/Metabolosome.shape"
```

Comment:
I really think it would be clearer if nitrogenase had its own file, even if it is literally just a duplicate of metabolosome but with a different name.